src/main/resources/CU-ApplicationResources.properties

```diff
@@ -527,3 +527,4 @@ message.concur.token.reset.success=The current Concur access token has been rese
 error.concur.token.retet.production=Resetting the access token to an emty string can not be done in a production environment.
 error.cbcp.batch.upload.email.subject=Error occurred during Corporate Billed Corporate Paid batch upload process.
 error.cbcp.batch.upload.email.body.starter=Errors occured during the Corporate Billed Corporate Paid upload process.
+error.create.accounting.document.attachment.download=Could not download the attachment from
```

Comment:
I would recommend adding a placeholder to this property, for consistency with similar messages in this file. If you need to resolve the placeholder programmatically, you can use the MessageFormat.format() method to do so.

Reply:
can you direct me to an example of what you mean hear, I am not sure I understand

Reply:
For an example, see the "error.required" property (in the Rice-side KRAD ApplicationResources.properties file) and the ConcurAccountValidationServiceImpl.checkRequiredAccountInfo() method.